02 - Web UI Template/CognitiveSearch.UI/Search/DocumentSearchClient.cs

```diff
@@ -370,33 +376,37 @@ private string[] GetContainerSasUris()
 
             string accountName = _configuration.GetSection("StorageAccountName")?.Value;
             string accountKey = _configuration.GetSection("StorageAccountKey")?.Value;
-
-            SharedAccessBlobPolicy adHocPolicy = new SharedAccessBlobPolicy()
-            {
-                SharedAccessExpiryTime = DateTime.UtcNow.AddHours(24),
-                Permissions = SharedAccessBlobPermissions.Read
-            };
-
+            StorageSharedKeyCredential storageSharedKeyCredential = new StorageSharedKeyCredential(accountName, accountKey);
             containerAddresses[0] = _configuration.GetSection("StorageContainerAddress")?.Value.ToLower();
```

Comment:
Does containerAddresses need to be a static field? If so, it should be named as `s_containerAddresses`